src/ipvs.c

```diff
@@ -302,8 +322,16 @@ static void cipvs_submit_service (struct ip_vs_service_entry *se)
 	cipvs_submit_if (pi, "if_packets", NULL, stats.inpkts, stats.outpkts);
 	cipvs_submit_if (pi, "if_octets", NULL, stats.inbytes, stats.outbytes);
 
+	size_t live_dests = 0;
```

Comment:
At your option: since you report "active" below, this variable might be better named "active_dests".